.github/workflows/e2e.yaml

```diff
@@ -12,7 +12,9 @@ on:
       - 'go.sum'
       - 'Dockerfile'
   pull_request:
-    branches: master
+    branches:
```

Comment:
I dont think the branches for `push` or `pull_request` need changing here. I think the bug here is that changes to `'e2e/**'` aren't triggering e2e workflow on pull request, probably due to copy and paste on original change. I think if that path is added to push/PR paths then this will trigger on this branch.

Reply:
@mcavoyk you're right. Added the path you suggested and undid my branches change. 

✅ e2e tests ran and passed